### PR TITLE
[#2908] Support for RTS options in docker run limited to -e NETWORK

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

File: nix/docker/context/bin/run-node [hunk 1]
```diff
@@ -36,6 +36,11 @@ if [[ -z $CARDANO_PORT ]]; then
   CARDANO_PORT=3001
 fi
 
+if [[ -z $CARDANO_RTS_OPTS ]]; then
+  # See: nix/nixos/cardano-node-service.nix#L401
```
Comment (drobakowski) [inline nix/docker/context/bin/run-node:40]:
```suggestion
  # See: nix/nixos/cardano-node-service.nix#L512
```
The position was updated in the meantime...

**Comment by drobakowski** [comment]
Any news on this? #2908

**Comment by tdiesler** [comment]
You can use [nessus-cardano](https://github.com/tdiesler/nessus-cardano) while we're waiting for this to show up upstream.

**Comment by disassembler** [comment]
we pass `$@` so RTS options or any other flags can be passed at runtime. Is there a reason that doesn't work for your use case?

**Pull request closed without merging**